examples/platform/silabs/display/lcd.h

```diff
@@ -33,6 +33,22 @@ class SilabsLCD
 {
 
 public:
+    typedef enum screen
+    {
+        DemoScreen = 0,
+        StatusScreen,
+        QRCodeScreen,
+        InvalidScreen,
+    } Screen_e;
+
+    typedef struct dStatus
+    {
+        uint8_t nbFabric     = 0;
+        bool connected       = false;
+        char networkName[50] = { "TODO" };
+        bool advertising     = false;
+    } DisplayStatus_t;
```

Comment:
Do we have to maintain a status structure? or can rather we fetch them with the different apis when we have to update the LCD